Bottom: 0ae11f2f8cfd858a36140403b84ba73ebc0b220a
Top:    602e58c1c3c48da107f41ef7ba945e33b9cbe159
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-02-07 14:38:32 +0530

Refresh of sched-fair-paraphrase

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 492d327baf92..d3b87a4b48e5 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5941,7 +5941,7 @@ static int select_idle_sibling(struct task_struct *p, int prev, int target)
 	if (recent_used_cpu != prev &&
 	    recent_used_cpu != target &&
 	    cpus_share_cache(recent_used_cpu, target) &&
-	    is_idle_task(recent_used_cpu) >= cpu_non_preempted_idle &&
+	    is_idle_cpu(recent_used_cpu) >= cpu_non_preempted_idle &&
 	    cpumask_test_cpu(p->recent_used_cpu, p->cpus_ptr)) {
 		/*
 		 * Replace recent_used_cpu with prev as it is a potential
